### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by AzizaAlSayed**

### CODE REVIEW

**Comment by laraSaidSayedAhmad** [comment]
I am going to add hash the passwords

**Comment by AzizaAlSayed** [comment]
I am going to add hash the passwords

File: app/root.tsx [hunk 1]
```diff
@@ -1,178 +1,42 @@
...
   return (
     <html lang="en">
       <head>
         <meta charSet="utf-8" />
-        <meta name="viewport" content="width=device-width,initial-scale=1" />
-        {title ? <title>{title}</title> : null}
-        <Meta />
-        <Links />
+        <title>Remix: So great, it's funny!</title>
+        <link rel="stylesheet" href="//demo.productionready.io/main.css"></link>
```
Comment (Abdallatif) [inline app/root.tsx]: you should define the links in the main route `app/routes/index.tsx`
```
export const links: LinksFunction = () => {
  return [{ rel: "stylesheet", href: "//demo.productionready.io/main.css" }];
};

Then in the Document you should keep rendering the `meta` & `links`
```
        <Meta />
        <Links />
```

File: prisma/seed.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    getUsers().map((user) => {
+      return PrismaDb.user.create({ data: user });
+    })
+  );
+}
+
+seed();
+
+function getUsers() {
+  // shout-out to https://icanhazdadjoke.com/
```
Comment (Abdallatif) [inline prisma/seed.ts]: ?

File: app/routes/register.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  ) {
+    throw new Error(`Form not submitted correctly.`);
+  }
+
+  const fields = { userName, password, email };
+
+  const userInfo = await db.user.create({
+    data: fields,
+  });
+  console.log(userInfo);
```
Comment (Abdallatif) [inline app/routes/register.tsx]: no console logs

File: app/root.tsx [hunk 4]
```diff
@@ -1,178 +1,42 @@
-import {
```
Comment (Abdallatif) [inline app/root.tsx]: I don't agree with most of the changes in this file. I still think we need `links`, `App`, `ErrorBoundary` & `CatchBoundary`.

**Review by Abdallatif — CHANGES_REQUESTED** [review]

**Review by Abdallatif — APPROVED** [review]

**Pull request merged**